.github/workflows/publish-images.yaml

```diff
@@ -38,10 +43,15 @@ jobs:
           push: true
           tags: ${{ steps.image-meta.outputs.tags }}
           labels: ${{ steps.image-meta.outputs.labels }}
+          sign_image: true
```

Comment:
Sign every image? Or maybe only when we build a release?

Reply:
I was thinking we should sign everything. I think its best not to do "special things" on release, unless we have to.